### ISSUE TITLE
composer.json and composer.lock are out of sync

### ISSUE THREAD

**Issue by mjordan** [mjordan/riprap#55]

**Comment by mjordan** [comment]
PHPUnit tests are failing due to deprecation warnings:

```
OK (6 tests, 17 assertions)

Remaining indirect deprecation notices (4)

  1x: The "Symfony\Bridge\Monolog\Processor\DebugProcessor::getLogs()" method will require a new "Request|null $request" argument in the next major version of its parent class "Symfony\Component\HttpKernel\Log\DebugLoggerInterface", not defining it is deprecated.
    1x in CheckFixityCommandTest::testExecute from App\Tests\Command

  1x: The "Symfony\Bridge\Monolog\Processor\DebugProcessor::countErrors()" method will require a new "Request|null $request" argument in the next major version of its parent class "Symfony\Component\HttpKernel\Log\DebugLoggerInterface", not defining it is deprecated.
    1x in CheckFixityCommandTest::testExecute from App\Tests\Command

  1x: The "Symfony\Bridge\Monolog\Logger::getLogs()" method will require a new "Request|null $request" argument in the next major version of its parent class "Symfony\Component\HttpKernel\Log\DebugLoggerInterface", not defining it is deprecated.
    1x in CheckFixityCommandTest::testExecute from App\Tests\Command

  1x: The "Symfony\Bridge\Monolog\Logger::countErrors()" method will require a new "Request|null $request" argument in the next major version of its parent class "Symfony\Component\HttpKernel\Log\DebugLoggerInterface", not defining it is deprecated.
    1x in CheckFixityCommandTest::testExecute from App\Tests\Command
```

**Comment by mjordan** [comment]
Resolved in #61.
